README.md

```diff
@@ -1,21 +1,43 @@
 # DevOps Apprenticeship: Project Exercise
```

Comment:
There is no documentation on how to the run the docker containers here (this is a minimum requirement for this to pass review)

Reply:
Instructions added to run docker containers